src/components/OrderedList/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * @file Ordered List entry point.
```

Comment:
Small sentence case suggestion:
```suggestion
 * @file Ordered list entry point.
```